bool gripper_state
---
bool success
